action.yml

```
@@ -0,0 +1,91 @@
+name: 'silverbackltd-build-action'
```

Can this have spaces in it?
```suggestion
name: 'Silverback Build and Push Images'
```